fem/fespace.cpp

```diff
@@ -1358,6 +1358,46 @@ int FiniteElementSpace::GetNConformingDofs() const
    return P ? (P->Width() / vdim) : ndofs;
 }
 
+int FiniteElementSpace::GetVectorDim() const
+{
+   const FiniteElement *fe;
+   if (!GetNE())
+   {
+      static const Geometry::Type geoms[3] =
+      { Geometry::SEGMENT, Geometry::TRIANGLE, Geometry::TETRAHEDRON };
+      fe = FEColl()->FiniteElementForGeometry(geoms[GetMesh()->Dimension()-1]);
+   }
+   else
+   {
+      fe = GetFE(0);
+   }
```

Comment:
I believe this logic (and similarly in `GetCurlDim`) could be simplified using PR #4260. @v-dobrev, should we try to move that PR forward?

Reply:
Moved to https://github.com/mfem/mfem/pull/4598#issuecomment-2524280095